src/components/panels/collapsible-panel/collapsible-panel.styles.js

```diff
@@ -64,13 +64,19 @@ const getHeaderContainerStyles = ({ isDisabled, isOpen, isSticky, theme }) => {
   ];
 };
 
-const getHeaderStyles = ({ isDisabled, isCondensed }) => {
+const getHeaderStyles = ({
+  isDisabled,
+  isCondensed,
+  headerControlsAlignment,
+}) => {
   const baseStyles = css`
     display: flex;
     flex: 1;
     align-items: center;
     list-style-type: none;
-    justify-content: space-between;
+    justify-content: ${headerControlsAlignment === 'left'
```

Comment:
I like the fact that alignment encapsulates the fact that flexbox is used instead of exposing all `justify-content` options through the prop.